app/code/Magento/Cms/Controller/Adminhtml/Wysiwyg/Images/DeleteFiles.php

```diff
@@ -62,6 +62,7 @@ public function execute()
     {
         try {
             if (!$this->getRequest()->isPost()) {
+                //phpcs:ignore Magento2.Exceptions.DirectThrow
```

Comment:
Lets move this check out of try catch and return the error resultJson befor the execution of the main code if the request is not post instead of adding two phpcs ignores